Library/Homebrew/shims/super/ninja

```diff
@@ -11,9 +11,16 @@ fi
 
 source "${HOMEBREW_LIBRARY}/Homebrew/shims/utils.sh"
 
+parallel_args=()
+integer_regex='^[0-9]+$'
+if [[ -n "${HOMEBREW_MAKE_JOBS}" ]] && [[ ${HOMEBREW_MAKE_JOBS} =~ ${integer_regex} ]]
```

Comment:
Maybe we make this error out in `determine_make_jobs` rather than silently ignoring non-integers?

Reply:
`determine_make_jobs` should never return a non-integer. This is really a guard against some (third-party) formula setting `HOMEBREW_MAKE_JOBS` directly to some non-integer.

Fine to let `ninja` complain about it instead, but I'd personally rather not hear from anyone who sets `HOMEBREW_MAKE_JOBS` and has a broken build as a consequence.

Reply:
We probably should have a cop banning `ENV["HOMEBREW_*"] = ...` in formulae to be honest. It's private API hacking 99% of the time.

Setting `HOMEBREW_MAKE_JOBS ` won't work properly anyway because it doesn't set `MAKEFLAGS`.

Reply:
Ok, let's just skip the integer check.